### DISCUSSION TITLE
user-guide/faqs/word-addon

### DISCUSSION THREAD

**Discussion by giscus** [zotero-chinese/wiki#122] [文档评论区]
# user-guide/faqs/word-addon

本页面整理了一些使用字处理插件常遇到的一些问题，你可以在右侧大纲栏（部分窄屏设备也可能在左侧）快速定位到你需要的内容。 如果没有在本页面找到你需要的问题，可以参考官方文档 Zotero 文字处理插件报告的一些问题以及可能的解决方案 (https://www.zotero.org/support/wordprocessorplugin_troublesh...

https://zotero-chinese.com/user-guide/faqs/word-addon.html

**Comment by QingbaoLi** [comment]
最近碰到Zotero（更新到7beta）在Word中出现“更新文档信息出错”这个错误，用了Zotero官方文档里的方法没用。最后是今天把Zotero以Win8兼容模式打开，解决问题。

**Comment by Jeremy-C** [comment]
想请问下，如果想在Zotero中实现 引注 的 上标 和 非上标 混排，有什么办法实现吗？（需求与http://forum.inoteexpress.com:60080/forum.php?mod=viewthread&tid=57863&extra=page%3D1和https://muchong.com/html/201006/2155956_2.html的一样）

**Reply by northword** [reply]
据我所知实现不了，我写小论文都是没管这个的，reviewer 和 editor 也没说啥。

**Comment by chaiquan12345** [comment]
您好，在不同片段引用同一本书或者同一篇文章的时候，会显示参同前注，如何实现精准定位到参同前面的哪一个注释呢？

**Reply by QingbaoLi** [reply]
有个插件，Zoterocitation，可以一试

**Reply by northword** [reply]
不是插件，这个与 csl 样式有关，建议去 csl 仓库问问https://github.com/redleafnew/Chinese-STD-GB-T-7714-related-csl

**Comment by ying13a** [comment]
如何用zotero分章节插入参考文献

**Reply by ying13a** [reply]
您好请问如何实现这个功能呢

**Reply by northword** [reply]
zotero 尚无法实现，可以把不同章节拆分到不同文件。

**Comment by SihuiZhu0927** [comment]
您好，请问在word中引用参考文献时最后的References顺序完全是打乱的是什么原因呢？

**Reply by SihuiZhu0927** [reply]
引文的顺序和参考书目的顺序不一致，参考书目的顺序完全是乱的

**Reply by northword** [reply]
用的哪个样式，不一致的截个图看看？

**Reply by SihuiZhu0927** [reply]
inline样式是(作者，年份)，引文的样式文献前面没有[1]这种序号，对话框好像发不了截图，不一致指的是引文里文献的前后顺序与文中不一致。

**Reply by northword** [reply]
样式名是什么，有的样式确实是特意不一致的（比如参考文献表按标题或作者字母排序的，按语言排序的等等）。

截了图后在输入框按 ctrl+c ，等待上传完，就可以发图片。

**Reply by SihuiZhu0927** [reply]
样式名是Bioinformatics, 又去看了一下该期刊的文章，确实是按照作者字母排序的，谢谢您。

**Comment by Lei-95** [comment]
添加参考文献（四个作者）后，发现参考格式显示了两个作者“one, two, et al.”，正常情况下应该是“one, et al.”，请问怎么解决

**Reply by northword** [reply]
取决于你是用的 csl 样式。

**Reply by Lei-95** [reply]
其他的都是对的，只有这个出现了问题

**Reply by northword** [reply]
哪个样式以及对应的条目数据是什么？

出现的地方是正文引注还是参考文献表？这个文献条目的题录信息准不准确？作者字段里面有没有出现一个作者字段填了两个作者的情况？

**Reply by Lei-95** [reply]
样式是 Geophysical Research Letters, 文献的DOI是10.1073/pnas.2310584120，文献信息是从官网下载的，作者字段的信息也是对的。在正文引注里出现了问题。

**Reply by northword** [reply]
![image](https://github.com/user-attachments/assets/f5237c2f-742e-49f7-bdb5-fcf6766b6081)

我这边正常的

![image](https://github.com/user-attachments/assets/27af8e60-88b0-45bd-bb4d-d72b3d82e64d)

**Reply by Lei-95** [reply]
这就很奇怪了

**Reply by northword** [reply]
删掉重新插入一下试试。或者也有可能是消岐导致的 -- 其他地方有一个和这个一样一作（名或姓）和年代的引注，为了消除歧义，这个显示了第二个作者。

**Reply by northword** [reply]
系消岐：

![image](https://github.com/user-attachments/assets/6272cace-51af-4ffb-bce0-b2d95a3e8b07)

https://docs.citationstyles.org/en/stable/specification.html#disambiguation

![image](https://github.com/user-attachments/assets/7b257405-14c7-46ca-b393-283b8ff2559a)

**Reply by Lei-95** [reply]
应该是消岐的问题。请问我需要在哪里修改呢？第一张图片的内容在哪里找？

**Reply by northword** [reply]
如果是消岐，那不需要修改，是正确的。

第一张图，设置-引用-样式编辑器。或直接以文本方式打开样式文件查看源码。

**Reply by Lei-95** [reply]
好的，多谢了！但是我在样式编辑器里找不到新添加的样式，刷新了也不行，还是只有zotero自带的那些样式。如果要以文本方式打开样式文件，需要怎么查找文件路径呢？

**Reply by northword** [reply]
https://zotero-chinese.com/user-guide/backup#%E6%95%B0%E6%8D%AE%E6%96%87%E4%BB%B6%E4%BD%8D%E7%BD%AE

这个目录下的 styles 文件夹。

你提到的这个样式是个从属样式（ https://zotero-chinese.com/csl-dev-guide/primer#%E7%90%86%E8%A7%A3-csl-%E6%A0%B7%E5%BC%8F ） ，你需要对应查找到它依赖的父样式 (https://www.zotero.org/styles/american-geophysical-union?source=1) 。

**Reply by Lei-95** [reply]
感谢详尽的解答！

**Comment by AngelSakura** [comment]
请问一下，我之前通过zotero插入的文献，现在再打开继续插入文献发现无法识别以前的引注。就是说我从文中某个位置再插入一篇文献会从1开始，无法和之前已经插入的文献相关联，也无法更新。

**Reply by northword** [reply]
确保你原来插入的没有unlink，如果点过unlink，需要在原始版本上继续编辑。

**Reply by AngelSakura** [reply]
您好，以前插入的没用点过unlink

**Reply by AngelSakura** [reply]
在原始版本上继续编辑的意思是要全部重新插入文献吗

**Reply by northword** [reply]
在以前的引注上右击，是否有「切换域代码」选项？没有的话，你可能已经点过unlink了，如果有的话，不应该出现这种情况的。

**Reply by northword** [reply]
参考：https://www.zotero.org/support/kb/existing_citations_not_detected

**Reply by AngelSakura** [reply]
好的，非常感谢

**Comment by sherkevin** [comment]
我想问一下每次导出文献列表到 word 或者 wps 时都变成了文本格式，而不是对应的列表格式，还要手动转换，这个可以解决吗

**Comment by zy15380228052** [comment]
引注选项窗口的「page」处有个下拉列表，里面是不同的定位符类型,这个page 在哪里啊 找不到蛮

**Comment by MarsshiyuY** [comment]
想请问一下easier citation插件的快捷引用按键无法使用（0.4.6版本）该怎么解决

**Comment by leavesice** [comment]
各位大佬好，最近遇到的问题是Zotero更新到7beta后，发现在word里面点击引用参考文献后，word菜单栏变成灰色并且无法跳转Zotero，尝试了各种办法（比如重新安装word加载项，以管理员身份运行，勾选兼容模式等）仍然无法解决该问题，请问现在该怎么处理

**Comment by ljyTota** [comment]
最近使用zotero（7）在word中引用时，在保持源分类搜索时，只能在打开的文件中搜索，导致需要没打开但是我建立条目的文件搜索不到，只能到经典视图添加

**Comment by shatanshangdechuan** [comment]
在zotero7中插入引注时，我需要的是[1],[2,[3],[4],[5],而非[1-5]，但是我在一个一个引用时，后面的文献会将前面的文献替代，我在设置中并没有找到与此有关的选项

**Comment by JamesSmith20** [comment]
请问有没有这么一种功能，把中文参考文献翻译为英文，并且在引用时标注(in Chinese)？如果不行，应该如何实现？非常感谢！

**Comment by Lei-95** [comment]
你好，我想在英文论文中引用中文专利，请问如何转成英文？

**Comment by Lei-95** [comment]
你好，请问zotero 怎么在英文论文里面引用中文专利呢？

**Comment by zy15380228052** [comment]
中文专利？我不太会 你看看网上有没有答案呢



发自我的iPhone


------------------ Original ------------------
From: Lei-95 ***@***.***&gt;
Date: Tue,Jan 14,2025 3:30 PM
To: zotero-chinese/wiki ***@***.***&gt;
Cc: zy15380228052 ***@***.***&gt;, Comment ***@***.***&gt;
Subject: Re: [zotero-chinese/wiki] user-guide/faqs/word-addon (Discussion#122)





 
你好，请问zotero 怎么在英文论文里面引用中文专利呢？

**Comment by zy15380228052** [comment]
好像没有？我不太确定 你去zotero中文社区看看呢



发自我的iPhone


------------------ Original ------------------
From: JamesSmith20 ***@***.***&gt;
Date: Wed,Dec 18,2024 11:06 AM
To: zotero-chinese/wiki ***@***.***&gt;
Cc: zy15380228052 ***@***.***&gt;, Comment ***@***.***&gt;
Subject: Re: [zotero-chinese/wiki] user-guide/faqs/word-addon (Discussion#122)





 
请问有没有这么一种功能，把中文参考文献翻译为英文，并且在引用时标注(in Chinese)？如果不行，应该如何实现？非常感谢！

**Reply by Lei-95** [reply]
好的，我看也没有相关的解决办法，多谢啦

**Comment by zhuyuhua23** [comment]
如何将论文中引用的部分改为其它颜色，如（zhu et al., 2001）括号里面设置为蓝色？以及如何直接查看括号里面的文献是哪篇？鼠标在上面空直接显示和编辑吗？插件可以帮忙改进吗？谢谢。

**Comment by xiaowu20** [comment]
请问打开zotero，在word中选择add/edit citation没有反应，但是选择Troubleshooting mode（以故障排除模式重启后）下回到word中就可以使用add/edit citation；然后退出Troubleshooting mode后，常规模式下又不能引用了，这是为什么？

**Reply by northword** [reply]
插件导致，更新所有插件和 Zotero，优先排除 easier citation 插件。

**Comment by lan-a-pi** [comment]
请教一下从zotero6升级到7，Add/Edit Citation正常，到显示插入文献时点Add/Edit Bibliography会弹出一个叫编辑参考文献表的框，点击ok后没有显示文献怎么解决。求求了在写毕业论文很需要解决这个问题

**Comment by krystal-Maker** [comment]
你好，我在修改参考文献时，弹出您已经修改该引文，是否保留更改？点击了是,导致引文无法更新，我按照上图的方法修改引文域代码，点击word文档中引文上标，右击显示为域代码，可是并没有在代码中找到"dontUpdate": true,这项。请问这是怎么回事呢？超级希望能回答此条，万分感谢！！！

**Reply by krystal-Maker** [reply]
请作者看看我！！！拜托拜托！！!

**Reply by northword** [reply]
删除该处引注重新插入。

**Comment by 1444582110** [comment]
为何 [100] 和 [99] 后的空隙不一样，按上面的操作并不能解决

**Comment by YangJJ1999** [comment]
添加原文的条目笔记后，会自带引用，但是英文文献的引用为什么还是“等”啊，在word中此引用是正常的“et al”，就是条目笔记中所有英文文献都是“等”，已确保Info信息是en

**Reply by northword** [reply]
印象中，条目笔记的引文样式不支持双语，而是根据界面语言确定。

**Comment by 1944640363** [comment]
尊敬的作者您好，为什么我的引文显示存在以下问题：对于支持双语混排的样式，有的样式能够按照双语混排（Arkhangelsky el al，2021；Brown and Clark，1958）;有的不能，例如（Arkhangelsky等，2021；Brown和Clark，1958）。

**Reply by northword** [reply]
检查语言 language 字段。

**Reply by 1944640363** [reply]
谢谢您的回复。语言字段没有问题，否则不会有的样式能够正常混排，例如（Arkhangelsky el al，2021；Brown and Clark，1958）。
我查看了不能混排的cls文件的源码，发现少了一个 default-locale="en-US"。补充之后问题就解决了。

**Comment by peter-hu-up** [comment]
请问，在zotero7中，我在word中插入文献时，多了a,b,c等，在参考文献那里也出现了三次同样的文献，这个该怎么解决啊。（在zotero中已经安装了linter插件来去除重复文献）

**Comment by YozhenMa** [comment]
请问一下大家，为什么我在zotero文献格式库中搜索到的“Energy”期刊的引用格式后进行引用，文中使用的是例如[3–5]这种格式，但是我去期刊网站查询已发表期刊后发现其使用的格式是例如[[3], [4], [5]]这种分开写的格式，这是为什么？按理来说下载的Energy期刊的文献格式应该与其发表的文章格式一致，辛苦大家解答一下，不知道该如何调整为和官网一致的格式。
